docs/UserGuide.md

```diff
@@ -120,6 +123,9 @@ Detailed explanations of each commands and their respective usage are listed und
 For example, if the command specifies `edit MODULE_CODE c/NEW_MODULE_CODE t/core`, `edit MODULE_CODE t/core c/NEW_MODULE_CODE
 ` is also acceptable.
 
+* When a command does not require parameters, e.g. `help`, adding extraneous words, e.g `help me` will not affect the
+ command.
```

Comment:
MINOR MINOR nit: should we add an extra, "allowing the command to run as per normal" elaboration to make it clearer?